regression/test.pl

```diff
@@ -107,7 +119,7 @@ ($$$$$)
           my $r;
           $result =~ s/\\/\\\\/g;
           $result =~ s/([^\\])\$/$1\\r\\\\?\$/;
-          system("bash", "-c", "grep \$'$result' \"$name/$output\" >/dev/null");
+          system("bash", "-c", "grep $grep_options \$'$result' \"$name/$output\" >/dev/null");
```

Comment:
test.pl does have --help output explaining the format it consumes - could you please amend that as well to include information about your added feature?